java/src/org/openqa/selenium/OutputType.java

```diff
@@ -55,7 +55,7 @@ public String toString() {
   OutputType<byte[]> BYTES = new OutputType<byte[]>() {
     @Override
     public byte[] convertFromBase64Png(String base64Png) {
-      return Base64.getMimeDecoder().decode(base64Png);
+      return Base64.getDecoder().decode(base64Png);
```

Comment:
I am not familiar with the decoding details performance-wise. However, based on https://docs.oracle.com/javase/8/docs/api/java/util/Base64.html#basic, they are to use different base64 encoding schemes. My concern is that it should not cause any unexpected errors.

Reply:
The WebDriver specification tells the base64 encoded data should be rfc4648 compliant.
(see the "Base64 Encode" link in https://www.w3.org/TR/webdriver/#index)

The `Base64.getMimeDecoder()` is the decoder to rfc2045 encoded data.
The `Base64.getDecoder()` is the decoder to rfc4648 encoded data.

So i think the usage of `Base64.getDecoder()` should be the correct way to decode the data.

Reply:
That sounds right. Appreciate the thoroughness! Thank you so much.